### ISSUE TITLE
Relayed: Concerning Station names 

### ISSUE THREAD

**Issue by RocketD0g** [openaq/project-universal-stationID#8]
This comment is relayed and originally from @maschu09. This comment is relayed and originally from @maschu09 (7+ 8 of 8 to transport over to GH):

> * station name can be any unicode string describing the station location.
> The length of station name should not exceed 200 characters.
> 
> * if a station name is given, it should remain constant throughout time.
> However, this is a weak requirement, because the identification of a station 
> is made via the station id.

**Comment by RocketD0g** [comment]
On station names (currently designated as ``location`` in [our data format](https://github.com/openaq/openaq-data-format), I think it might be truest and most transparent to the originating source to keep the name as-is. 

Currently the names (or the ``location`` parameter) are not picked by us, but rather designated as how the originating source presents it. This allows some transparency for a given agency to understand which station is which on our system, while the new `stationID` type parameter will allow a universal, persistent method of identifying the station, no matter if the originating source changes the name. Does this make sense? (Especially to @maschu09 and @jflasher)

**Comment by jflasher** [comment]
Makes sense to me. Put another way, I am not sure how we'd come up with something different than both `stationID` and `location` to use as a station name.

**Comment by maschu09** [comment]
@RocketD0g you captured exactly what I intended to say: station name shoul dremain under full control of the originating agency/data provider. The two suggestions I made were only meant to cast this into some very loose rules that can be used for creating databases and software that makes use of the `location` parameter, i.e. fixing a length and data type.
